xbmc/video/VideoDatabase.cpp

```diff
@@ -6271,8 +6310,8 @@ bool CVideoDatabase::GetYearsNav(const std::string& strBaseDir, CFileItemList& i
       }
       else if (idContent == VIDEODB_CONTENT_MUSICVIDEOS)
       {
-        strSQL = "select musicvideo_view.premiered, count(1), count(files.playCount) from musicvideo_view ";
-        extFilter.AppendJoin("join files on files.idFile = musicvideo_view.idFile");
+        strSQL = "SELECT musicvideo_view.premiered, COUNT(1), COUNT(history.idFile) FROM musicvideo_view ";
+        extFilter.AppendJoin("JOIN history ON history.idFile = musicvideo_view.idFile");
```

Comment:
Check if we can use subqueries here instead, as inner join might hide some values